### ISSUE TITLE
Remove robot build as part of pipeline

### ISSUE THREAD

**Issue by cmungall** [geneontology/pipeline#12]
https://github.com/ontodev/robot/issues/211

**Comment by dougli1sqrd** [comment]
This is blocked on https://github.com/ontodev/robot/issues/211 still. I think all we need to do is make a release of robot into sonatype/maven

**Comment by kltm** [comment]
Plus 1) add in the variables to the pipeline, 2) get the downloader setup, and 3) added into either the step bin or the "global" bin.

**Comment by kltm** [comment]
After a little discussion with @cmungall and @dougli1sqrd , we've decided that while important this is not a pressing need for the minimal released pipeline.

**Comment by cmungall** [comment]
Shall we switch the ontology pipeline to use the osk Docker image?

https://github.com/INCATools/ontology-starter-kit/issues/22

**Comment by kltm** [comment]
If the tool is being called on the command line from the Makefile, I'm not sure how that would make things easier unless it is dropping the bin on disk. We could run the Makefile from within the docker environment, but I think that would probably be more confusing than what we have now.

**Comment by kltm** [comment]
Is ROBOT used anywhere besides the go-ontology build ontology step? If not, this is superseded by #76 and can be closed.

**Comment by cmungall** [comment]
Check with Eric, may be used to prep data for blazegraph

**Comment by dougli1sqrd** [comment]
Yes, it seems we no longer use robot in the megastep. I forget what we used to use it for. There are no references to robot in the makefile.

**Comment by kltm** [comment]
Hijacking ticket for removal--otherwise a dupe of #76
